frontend/src/components/login/EyeHideLogo.tsx

```diff
@@ -0,0 +1,39 @@
+interface EyeHideLogoProps {
+  width?: string;
+  height?: string;
+}
+
+const EyeHideLogo = ({ width = "100%", height = "100%" }: EyeHideLogoProps) => {
+  return (
+    <div style={{ width, height }}>
+      <svg
```

Comment:
Can consider move this svg to `public` folder as that folder is meant to store all the images/assets.

Reply:
@tryyang2001 I think in this case Weida has made the logo a component, so it cannot be placed inside `public` as the folder is meant to serve static assets only. 

@weidak I placed the PeerPrep logo in a component because I felt that it will be resized often throughout the pages so a component will be more convenient for reuse. However, I think we can consider treating svgs that are used only in limited pages/do not need to be resized often as just images instead and place them in the `public` folder to reduce the number of component files. What do you think?

Reply:
>  I think in this case Weida has made the logo a component, so it cannot be placed inside public as the folder is meant to serve static assets only.

Oh I thought the svg itself can be a static asset? We just import the image from the `public` folder to make our component. Does this make sense?

Reply:
> Oh I thought the svg itself can be a static asset? We just import the image from the `public` folder to make our component. Does this make sense?

Ok moved it to public, and replaced the logo components with <Image> that uses the svg in public folder.